src/lib/baseCommand.ts

```diff
@@ -152,17 +152,19 @@ export default class Command {
       {
         name: 'main',
         type: String,
-        description: 'Should this version be the primary (default) version for your project?',
+        description:
+          "Should this version be the primary (default) version for your project? (Must be 'true' or 'false')",
```

Comment:
Adding so much boilerplate into the help screens for "must be true or false" seems like a lot.

Reply:
yeah I don't love this but not sure if it's clear otherwise since the user is just supposed to pass in a string. ideally this should be a boolean but I've given these commands more love than they already deserve lol